src/lib/api/ticket.ts

```diff
@@ -67,6 +67,31 @@ export async function editTitle(id: Ticket['ticket_id'], title: Ticket['title'])
             throw new UnexpectedStatusCode(status);
     }
 }
+/** Assigns the `priority_id` field of a {@linkcode Ticket} to set ticket priority. Returns `false` if not found. */
```

Comment:
Just a nitpick here.
```suggestion

/** Assigns the `priority_id` field of a {@linkcode Ticket} to set ticket priority. Returns `false` if not found. */
```